client/apps/edit/components/section_header/index.coffee

```diff
@@ -0,0 +1,56 @@
+React = require 'react'
+moment = require 'moment'
+RichTextParagraph = React.createFactory require '../../../../components/rich_text/components/input_paragraph.coffee'
+{ div, p, textarea } = React.DOM
+
+
+module.exports = React.createClass
+  displayName: 'SectionHeader'
+
+  changeTitle: (e) ->
+    if e.key is 'Enter'
+      e.preventDefault()
+    @props.article.set 'title', this.refs.title.value
+    @props.saveArticle()
+
+  changeLeadParagraph: (html) ->
+    @props.article.setLeadParagraph(html)
+    @props.saveArticle()
+
+  getPublishDate: ->
```

Comment:
This feels like something that could live as a backbone model method on `Article`... BUT also a classic case where things get unclear in transitioning to a new React world from Backbone. [@damassi and I discussed this a while back](https://artsy.slack.com/archives/C0350895E/p1488494785987356) wrt Force.

_So don't change it, just wanted to link to that Slack convo_ 😁